<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Indicator</class>
 <widget class="QDialog" name="Indicator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>415</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>415</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>415</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Indicators</string>
  </property>
  <widget class="QScrollArea" name="available_indicators_area">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>210</width>
     <height>211</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="layerCustomizationWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>208</width>
      <height>209</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="available_label">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>10</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Available</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="current_indicators_area">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>210</width>
     <height>135</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="layerCustomizationWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>208</width>
      <height>133</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="current_label">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>250</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current</string>
   </property>
  </widget>
  <widget class="QPushButton" name="apply_button">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>380</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="properties_area">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>240</width>
     <height>341</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>0</height>
    </size>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="layerCustomizationWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>238</width>
      <height>339</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="properties_label">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>10</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Properties</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
